app/src/main/kotlin/org/vinaygopinath/launchchat/screens/main/MainActivity.kt

```diff
@@ -87,7 +92,14 @@ class MainActivity : AppCompatActivity() {
         if (phoneNumbers.isEmpty()) {
             phoneNumberInputLayout.error = getString(R.string.toast_invalid_phone_number)
         } else if (phoneNumbers.size != 1) {
-            // TODO Multiple phone numbers detected
+            showPhoneNumberSelectionDialog(phoneNumbers) { selectedNumber ->
```

Comment:
(Not specific to this line, just adding a comment so we can keep any responses in a threaded conversation) Could you please add instrumentation tests?